flyteplugins/go/tasks/pluginmachinery/flytek8s/copilot.go

```diff
@@ -172,7 +172,10 @@ func AddCoPilotToContainer(ctx context.Context, cfg config.FlyteCoPilotConfig, c
 	if c.SecurityContext.Capabilities == nil {
 		c.SecurityContext.Capabilities = &v1.Capabilities{}
 	}
-	c.SecurityContext.Capabilities.Add = append(c.SecurityContext.Capabilities.Add, pTraceCapability)
+	var add_sys_ptrace bool = false
+	if add_sys_ptrace {
```

Comment:
Isn't it always false here? Should we add `add_sys_ptrace` to the plugin config?

Reply:
If there is a larger conversation here, which I suspect there is. We can split this into multiple PRs - I don't think merging support for applying a PodTemplate to initContainers is controversial.

Reply:
>We can split this into multiple PRs
Agree. @vlibov thoughts?

Reply:
Hi @davidmirror-ops,
how do we want to proceed? Shall we split the PR or make the SYS_PTRACE configurable (or remove it completely?)
Thanks!

Reply:
>Shall we split the PR or make the SYS_PTRACE configurable
Hey @vlibov! Can we do both? Split this into two PRs: one for the PodTemplate for initContainer and the other making SYS_PTRACE configurable.

Reply:
Sounds good, @davidmirror-ops!
I reverted the SYS_PTRACE commit in this PR so it now only affects the initContainer part.

I will also create a new PR for the SYS_PTRACE config - can anybody point me on how to properly implement this? i.e. where is this option supposed to be defined and how to propagate it down to the relevant function which sets SYS_PTRACE? Thanks!

Reply:
Hey @davidmirror-ops, actually I created a new PR also for this change (initContainers)
https://github.com/flyteorg/flyte/pull/5664
I hope this is ok!
The #5556 can thus be closed...

Reply:
Hi @vlibov, just wondering did you end up creating a separate PR for removing SYS_PTRACE?